# the tray ID
uint32 id
---
# success of reservation
bool success
